extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/shared.properties.js

```diff
@@ -132,7 +133,47 @@ angular.module('PaperUI.controllers.rules').service('sharedProperties', function
 
         setModuleTypes : function(mTypes) {
             moduleTypes = mTypes;
+        },
+
+        areItemsDirty : function() {
+            return isDirty;
+        },
+        setItemsDirty : function(value) {
+            isDirty = value;
         }
 
     }
+});
+angular.module('PaperUI.constants').constant('itemConfig', {
+    'types' : [ {
+        name : "SwitchItem",
+        value : "Switch"
+    }, {
+        name : "ContactItem",
+        value : "Contact"
+    }, {
+        name : "StringItem",
+        value : "String"
+    }, {
+        name : "NumberItem",
+        value : "Number"
+    }, {
+        name : "DimmerItem",
+        value : "Dimmer"
+    }, {
+        name : "DateTimeItem",
+        value : "DateTime"
+    }, {
+        name : "ColorItem",
+        value : "Color"
+    }, {
+        name : "ImageItem",
+        value : "Image"
+    }, {
+        name : "PlayerItem",
+        value : "Player"
+    }, {
+        name : "LocationItem",
+        value : "LocationItem"
```

Comment:
The LocationItem is the only one that name and value is equal. The other values consists of the name without the leading "Item". Has this done by intention?